corehq/apps/accounting/invoicing.py

```diff
@@ -777,41 +783,44 @@ def _unit_description_by_user_type(self, user_type):
                 )
             )
         if self.quantity > 0:
-            return ngettext(
-                "Per-{user} fee exceeding limit of {monthly_limit} {user} "
-                "with plan above.{prorated_notice}",
-                "Per-{user} fee exceeding limit of {monthly_limit} {user}s "
-                "with plan above.{prorated_notice}",
-                self.rate.monthly_limit
-            ).format(
-                monthly_limit=self.rate.monthly_limit,
-                prorated_notice=prorated_notice,
-                user=user_type
+            return _(
+                f"Fee for each {user_type} exceeding the plan limit of {self.rate.monthly_limit}."
+                + prorated_notice
```

Comment:
I like the more concise wording, but now that we're passing in an untranslated `user_type`, I think we need to do the translation here:
```
return _('Fee for each {user_type} exceeding the plan limit of {monthly_limit}.').format(
  user_type=_(user_type), monthly_limit=self.rate.monthly_limit) + prorated_notice

Reply:
I get so mixed up with how translations work. Does translating as an f-string just work for the text that's static in it, then?